packages/button/src/Button.story.mdx

```diff
@@ -0,0 +1,206 @@
+
+<Meta title="Form|Button" />
+
+# Farmblocks Button
+
+A React component for displaying buttons and actionable icons
+
+## Basic Example
+
+```jsx
+import Button, {buttonVariants, PrimaryButton} from "@crave/farmblocks-button";
+const Form = () => (
+  <Button onChange={handleButtonClick} variant={buttonVariants.PRIMARY} small>
+    Click me
+  </Button>
+
+  <PrimaryButton onChange={handleButtonClick} small>
+    Click me
+  </PrimaryButton>
+);
+```
+
+
+## Properties
+
+- **children** (_Node)_
+- **text** (_String_)
+  > Text to be rendered as the button's children. Children will be ignored when `text` is present
+- **onClick** (_Function_)
+  > Callback to listen to button clicks
+- **small** (_Boolean_)
+  > Set the button height to 32px. The default height is 40px
+- **active** (_Boolean_)
+  > Set the active style
+
+<Preview>
+  <Story name="Default">
+    <Button onClick={action('onClick')}>Default button</Button>
+  </Story>
+  <Story name="Small">
+    <Button onClick={action('onClick')} small>Small button</Button>
+  </Story>
+  <Story name="Active">
+    <Button onClick={action('onClick')} active>Active button</Button>
+  </Story>
+</Preview>
+
+- **disabled** (_Boolean_)
+  > Set the loading style
+
+<Preview>
+  <Story name="Disabled">
+    <Button onClick={action('onClick')} disabled>Disabled</Button>
+  </Story>
+</Preview>
+
+- **loading** (_Boolean_)
+  > Set the loading style
+
+<Preview>
+  <Story name="Loading">
+    <Button loading>Loading</Button>
+  </Story>
+</Preview>
+
+- **fluid** (_Boolean_)
+  > Set width to 100% of it's parent
+  
+<Preview>
+  <Story name="Fluid">
+    <div style={{ width: 400, height: 400, background: "rgba(0, 0, 0, 0.32)" }}>
+    <Button variant={buttonVariants.PRIMARY} fluid text="Primary" />
+    <br />
+    <Button variant={buttonVariants.PRIMARY} fluid icon={<MdSettings size={22} />}>
+      Primary With Icon
+    </Button>
+  </div>
+  </Story>
+</Preview>
+
+## Using Icons
+- **icon** (_Node_)
+  > farmblocks-icon node to be rendered on the left side, or as an icon only button
+- **rightIcon** (_Node_)
+  > farmblocks-icon node to be rendered on the right side
+
+<Preview>
+  <Story name="With icon">
+    <Button onClick={action('onClick')} icon={<MdSettings size={22} />}>Left Icon</Button>
+  </Story>
+  <Story name="With right icon">
+    <Button onClick={action('onClick')} rightIcon={<SmChevronDown size={20} />}>Right Icon</Button>
+  </Story>
+  <Story name="With both icons">
+    <Button onClick={action('onClick')} icon={<MdSettings size={22} />} rightIcon={<SmChevronDown size={20} />}>Both icons</Button>
+  </Story>
+</Preview>
+
+
+## Default Variants
+These are the default themes that this component works with. 
+
+- **variant** (_One of buttonVariants_) = `buttonVariants.NEUTRAL`
+  > set the variant style
+
+## Custom Theme
+It's possible to customize the themes using styled-components `ThemeProvider`. Please refer to the `themeStyle` file to see implementation details
```

Comment:
Did you plan to extract the `themeStyle` function to a new file? It's currently inside `styledComponents/Buttons.js`. Either do that or update this text.

Reply:
Ohh, good catch! I guess i've changed the file name (or just messed up the names 😅)
It should be the `buttonThemes` file.